lib/docker.js

```diff
@@ -162,40 +167,56 @@ Docker.prototype.checkAuth = function(opts, callback) {
  * @param {Function} callback Callback
  */
 Docker.prototype.buildImage = function(file, opts, callback) {
+  var pack = tar.pack();
+  var content;
+  var self = this;
+
   if (!callback && typeof opts === 'function') {
     callback = opts;
     opts = null;
   }
 
-  var self = this;
-  var optsf = {
-    path: '/build?',
-    method: 'POST',
-    file: file,
-    options: opts,
-    isStream: true,
-    statusCodes: {
-      200: true,
-      500: 'server error'
-    }
-  };
+  function build(file) {
```

Comment:
I didn't reformat it, I moved the code to `build` function and it's 2 spaces but not TAB